packages/react/src/components/hero/Hero.stories.tsx

```diff
@@ -0,0 +1,546 @@
+    '--diagonal-koros-position': args.diagonalKorosPosition,
+    '--horizontal-padding-small': args.horizontalPaddingSmall,
+    '--horizontal-padding-medium': args.horizontalPaddingMedium,
+    '--horizontal-padding-large': args.horizontalPaddingLarge,
+    '--horizontal-padding-x-large': args.horizontalPaddingXLarge,
+    ...args.theme,
+  };
+
+  const theme = Object.fromEntries(Object.entries(argsAsTheme).filter(([, value]) => !!value));
+  const heroProps: HeroProps = {
+    koros: args.koros,
+    theme,
+    imageSrc: imageFile,
+    variant: args.variant,
+  };
+  return (
+    <div>
+      <style>
+        {`
+        .oddly-padded {
+          padding: 20px ${demoPadding};
+          background:${demoBgColor};
+        }
+        .oddly-padded p{
+          max-width: var(--container-width-xl);
+          margin: 0 auto;
+        }
+        .theme {
+          padding: 20px 20px 20px ${demoPadding};
+          font-size:10px;
+          border:1px solid #000;
+        }
+      `}
+      </style>
+      <Hero {...heroProps}>
+        <DefaultContent />
+      </Hero>
+      <div className="oddly-padded">
+        <p>This text should align with the hero content box on all screen sizes</p>
+      </div>
+      <div className="theme">
+        <p>Applied theme:</p>
+        <pre>{JSON.stringify(theme, null, 2)}</pre>
+      </div>
+    </div>
+  );
+};
+
+PlaygroundForTheme.argTypes = {
+  ...getDisabledControl('centeredContent'),
+  ...getDisabledControl('imageSrc'),
+  ...getDisabledControl('koros'),
+  backgroundColor: {
+    defaultValue: demoBgColor,
+    control: 'color',
+    description: 'Background color. Also koros color, if not set.',
+  },
+  color: {
+    defaultValue: '',
+    control: 'color',
+    description: 'Text color.',
+  },
+  korosColor: {
+    defaultValue: '',
+    control: 'color',
+    description: 'Optional koros color. Default is "--background-color"',
+  },
+  imagePosition: {
+    defaultValue: '',
+    description:
+      'How image is aligned to its container when image is larger than the container. Value can be any valid value for css rule "object-fit"',
+    control: {
+      type: 'select',
+      options: [
+        'top left',
+        'top center',
+        'top right',
+        'center left',
+        'center center',
+        'center right',
+        'bottom left',
+        'bottom center',
+        'bottom right',
+        '',
+      ],
+    },
+  },
+  diagonalKorosPosition: korosPosition,
+  horizontalPaddingSmall: {
+    defaultValue: demoPadding,
+    control: 'text',
+    description: 'Horizontal padding on small screens <768px.',
+  },
+  horizontalPaddingMedium: {
+    defaultValue: demoPadding,
+    control: 'text',
+    description: 'Horizontal padding on medium screens >=768px.',
+  },
+  horizontalPaddingLarge: {
+    defaultValue: demoPadding,
+    control: 'text',
+    description: 'Horizontal padding on large screens >=992px.',
+  },
+  horizontalPaddingXLarge: {
```

Comment:
Add the notion about these being strictly for Storybook, not actual props.
Reminder: `description: 'Storybook control, not an actual component property.'`

Reply:
Or add something about this being an actual theme variable, like: `description: 'Storybook control for theme variable, not an actual component property.'` ?

Reply:
Ok

Reply:
New descriptions and summaries have been set.